### ISSUE TITLE
[Feature Request] Implement Merge Block Presets as interpolation method selections

### ISSUE THREAD

**Issue by stormwulfren** [bbc-mc/sdweb-merge-board#8]
Would be a really useful if I could select the UNET block merge presets from your other extension as options for inter-method.

Further, I'd really like to see an extension for the rebasin method of merging models, if that's doable?

**Comment by bbc-mc** [comment]
Hi thank you for request.

I want to figure out what you mean "presets" works how.
U-Net "Block merge" is suported in MBW/MBW Each.

and I try git-re-basin, but dont work at all. (all image gone after re-basin)
so trying to understand what is going on inside method in case my impl is wrong.
Just i'm confused...

**Comment by stormwulfren** [comment]
Yeah, basically what I'd like to be able to do is, within merge board be able to do use the Merge Block methods to merge.
Something like:

``` Example
# Final Model = Model A + Model B, MergeMethod, base_alpha,format
Merge1 = StableDiffusion1.5 + WaifuDiffusion1.3,GRAD_A,0.5,ckpt
Merge2 = StableDiffusion1.5 + WaifuDiffusion1.3,COSINE,0.07,ckpt
Final = __O1__ + __O2__,0.5,safetensors
```

As for rebasin, I've not got it to work either. I'm planning to take a look at the code at some point and have a go myself.
